resources/yki/localisations/fi.json

```diff
@@ -33,16 +33,16 @@
   "email.evaluation_payment_success.content.line3": "Jos osakokeen arviointi muuttuu tarkistusarvioinnissa, palautamme tarkistusarviointimaksun. Et voi valittaa tarkistusarvioinnin päätöksestä.",
   "email.evaluation_payment_success.content.line4": "Lisätietoja",
   "email.evaluation_payment_success.subject": "Yleiset kielitutkinnot (YKI), tarkistusarviointipyyntö",
-  "email.login.content.line1": "Olet ilmoittautumassa seuraavaan yleiseen kielitutkintoon (YKI)",
-  "email.login.content.line2": "Ilmoittaudu testiin tämän linkin kautta",
+  "email.login.content.line1": "Olet ilmoittautumassa seuraavaan YKI-testiin",
+  "email.login.content.line2": "Ilmoittaudu YKI-testiin tämän linkin kautta",
```

Comment:
Tää "YKI-" näkyy puuttuvan ruotsin ja engl. kielisistä käännöksistä. Pitäis ehkä olla niissä vaikkei virkailijat ookaan tuota puutetta huomanneet. Samalla lailla rivin 40 `taso -> taitotaso` ja `testi -> YKI-testi` muutos puuttuvat ruotsinkielisestä käännöksestä.

Vois mainita näistä ophlaisille kun luultavasti sellaisia muutoksia jotka haluttais toteuttaa.

Reply:
Joo, vois tosiaan vielä varmistaa käännökset näihin. Nää muutokset pyydettiin samassa Excelissä kuin uuden käyttöliittymän tekstimuutokset, siinä ei vielä mistään käännöksistä puhuttu.

Reply:
OPH tuottaa näihin käännökset myöhemmin. Mennään näillä tuotantoon.